libs/packages/sam-formly/src/lib/formly/formly.module.ts

```diff
@@ -0,0 +1,49 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ReactiveFormsModule } from '@angular/forms';
+import { FormlyModule } from '@ngx-formly/core';
+import { FormlySelectModule } from '@ngx-formly/core/select';
+import { FIELD_TYPE_COMPONENTS, FORMLY_CONFIG } from './formly.config';
+import {
+  SdsAccordionModule,
+} from '@gsa-sam/components';
+
+export function minlengthValidationMessage(err, field) {
+  return `Should have atleast ${field.templateOptions.minLength} characters`;
+}
+
+export function maxlengthValidationMessage(err, field) {
+  return `This value should be less than ${field.templateOptions.maxLength} characters`;
+}
+
+export function minValidationMessage(err, field) {
+  return `This value should be more than ${field.templateOptions.min}`;
+}
+
+export function maxValidationMessage(err, field) {
+  return `This value should be less than ${field.templateOptions.max}`;
```

Comment:
Add comments for function and why are these just exported in the global namespace?

Reply:
These are some basic validations for all formly types not specific to individual types.